<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsDialog</class>
 <widget class="QDialog" name="settingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="colorsGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Colors</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="previewColorsGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>GroupBox</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="minimumSize">
            <size>
             <width>112</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string>Background</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QFrame" name="backgroundColorFrame">
              <property name="frameShape">
               <enum>QFrame::WinPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="backgroundColorChooser">
              <property name="text">
               <string>Pick</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/res/color_pick</normaloff>:/res/color_pick</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="minimumSize">
            <size>
             <width>112</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string>Original Shapes</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QFrame" name="flashColorFrame">
              <property name="frameShape">
               <enum>QFrame::WinPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="flashColorChooser">
              <property name="text">
               <string>Pick</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/res/color_pick</normaloff>:/res/color_pick</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_7">
           <property name="minimumSize">
            <size>
             <width>112</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string>Edited Shapes</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QFrame" name="adjustmentColorFrame">
              <property name="frameShape">
               <enum>QFrame::WinPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="adjustmentColorChooser">
              <property name="text">
               <string>Pick</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/res/color_pick</normaloff>:/res/color_pick</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_8">
           <property name="minimumSize">
            <size>
             <width>112</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string>Lines</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QFrame" name="lineColorFrame">
              <property name="frameShape">
               <enum>QFrame::WinPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="lineColorChooser">
              <property name="text">
               <string>Pick</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/res/color_pick</normaloff>:/res/color_pick</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="uiColorsGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>GroupBox</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="minimumSize">
            <size>
             <width>112</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string>GroupBox</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QFrame" name="frame">
              <property name="frameShape">
               <enum>QFrame::WinPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="minimumSize">
            <size>
             <width>112</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string>GroupBox</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QFrame" name="frame_4">
              <property name="frameShape">
               <enum>QFrame::WinPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_4">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="minimumSize">
            <size>
             <width>112</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string>GroupBox</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="frameShape">
               <enum>QFrame::WinPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_2">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="minimumSize">
            <size>
             <width>112</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string>GroupBox</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QFrame" name="frame_3">
              <property name="frameShape">
               <enum>QFrame::WinPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_3">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>settingsDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
